src/features/app-interfaces/components/create-app-interface-form.tsx

```diff
@@ -141,26 +182,36 @@ function FormInner({ helper }: FormInnerProps) {
         field: 'name',
         label: 'Name',
       })}
-      <div className="flex flex-col sm:flex-row sm:items-start sm:gap-4">
-        {helper.numberField({
-          field: 'applicationId',
-          label: 'Application ID',
-        })}
-        <div className="h-10 content-center sm:mt-[1.375rem]">OR</div>
-        <div className="grid sm:mt-[1.375rem]">
-          {!deployButtonStatus.disabled && deployButton}
-          {deployButtonStatus.disabled && (
-            <Tooltip delayDuration={400}>
-              <TooltipTrigger asChild>
-                <div tabIndex={0}>{deployButton}</div>
-              </TooltipTrigger>
-              <TooltipContent>
-                <span>{deployButtonStatus.reason}</span>
-              </TooltipContent>
-            </Tooltip>
-          )}
+      {isArc4AppSpec(appSpec) && (
+        <div className="flex flex-col sm:flex-row sm:items-start sm:gap-4">
```

Comment:
I think you don't need this `div` here, it's for formatting the "OR deploy app" section and you don't have it here.